packages/parser/src/lib.js

```diff
@@ -86,7 +86,7 @@ function splice(cont, idx, rem, str) {
 }
 
 function getHeaders(content) {
-  const index = content.indexOf("---", 1) + "---".length;
+  const index = content.indexOf("---") < 0 ? 0 : content.indexOf("---", 1) + "---".length;
```

Comment:
If a MD file has no header section this line could cause breaking changes to the file. If the file has no header the slice index should be 0.